src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -80,12 +82,24 @@ class PopulationNearRoadsJob(
 
     // Each buffered road "overflowed", we need to join it back up, not going to trim it tough
   val roadMaskRdd: RDD[(SpatialKey, MutableArrayTile)] = {
+    // TODO: we include the buffer here, we need to convert every road to UTM ... THEN buffer it.
     roadsRdd
       .flatMap { case (_, roads) =>
         val buffered = roads
+          .map{ road => // project to UTM and count
+            val utmZoneCrs = {
+              val center = road.getCentroid.reproject(WebMercator, LatLng)
+              UTM.getZoneCrs(center.getX, center.getY)
+            }
+            val utmRoad = road.reproject(WebMercator, utmZoneCrs)
+            val included = roadFilter(utmRoad.data)
+            roadHistogram.add((utmRoad.geom.getLength(), included, utmRoad.data))
+            // Buffer by 2KM to either size and reproject to match raster source
+            val meters = 2000
+            utmRoad.mapGeom(_.buffer(meters).reproject(utmZoneCrs, layoutTileSource.source.crs))
```

Comment:
I had to inline the function here because now there is a use for the geometry as line to count the length and then as polygon, to represent the buffered mask.